docs/en/writing-running-appium/caps.md

```diff
@@ -33,6 +33,7 @@
 |`androidCoverage`| Fully qualified instrumentation class. Passed to -w in adb shell am instrument -e coverage true -w | `com.my.Pkg/com.my.Pkg.instrumentation.MyInstrumentation`|
 |`enablePerformanceLogging`| (Chrome and webview only) Enable Chromedriver's performance logging (default `false`)| `true`, `false`|
 |`androidDeviceReadyTimeout`|Timeout in seconds used to wait for a device to become ready after booting|e.g., `30`|
+|`androidInstallTimeout`|Timeout in milliseconds used to wait for an apk to install to the device|e.g., `90000`|
```

Comment:
This should note the default value. See, for instance, the entry for `autoWebviewTimeout`